### ISSUE TITLE
Upgrade resets language to en-US

### ISSUE THREAD

**Issue by bnw42** [Eclipse-Community/r3dfox-old#66]
Just upgraded to v.126 (Win7 Sp1). Thanks! The only issue is that after the upgrade the default language had switched to en-US. Not a serious issue for me since I use en-GB & I was able to easily switch back to en-GB, but if I was using a language other than English it could be more problematic.

**Comment by the-r3dacted** [comment]
Is this upstream?

~~I really need to add a template.~~

**Comment by bnw42** [comment]
What do you mean by upstream in this context? Prior to the 126 upgrade the language was en_GB. I downloaded r3dfox-126.0.en-US.win64.installer.exe, ran it and selected the 'upgrade' option. When r3dfox started the language had reset back to en-US.

**Comment by the-r3dacted** [comment]
> What do you mean by upstream in this context?

Like install regular Firefox 125 on Windows 10 or 7 with VxKex, set language, then update to 126.

**Comment by bnw42** [comment]
Not using VxKex. I had r3dfox 125.03 installed on Win7 Sp1 64bit and installed/upgraded r3dfox 126.0 on top of that.

**Comment by the-r3dacted** [comment]
No, I'm saying to test regular 125 with VxKex, set language, and then upgrade to 126.

**Comment by jonm58** [comment]
r3dfox not autoupdate language pack

**Comment by bnw42** [comment]
I've tried VxKex but its support for current browser versions is limited and my anti-virus screamed loudly when I tried to run it, which disinclines me to persist with it.
I do have Firefox on a Win10 machine with the language also set as en-GB. It's currently running 125.03 which it insists is the most recent version.

**Comment by the-r3dacted** [comment]
Hmm, I would need to know if it happens with regular en-US build Firefox. However I don't really have any interest in supporting other languages because it adds a lot of complexity to test for and I don't know anything other than English so I might just do nothing anyways idk.

**Comment by bnw42** [comment]
Understand about additional languages making things more complex. As I said, it's not really an issue for me, I just changed it back when I noticed the language had changed. But it's not normal behaviour with firefox (or fork) updates, so thought I'd mention it. As for regular firefox 126, I'll know if that resets the language when FF on my win10 machine gets the 126 update. Of course, I always get the GB update with firefox whereas r3dfox only has the US version, so it's possible when a new r3dfox version is installed, it automatically upgrades to the en-US language. If so, resetting the language is probably unavoidable when upgrading r3dfox.
